.asf.yaml

```diff
@@ -35,6 +35,13 @@ github:
     dev:
       required_status_checks:
         strict: true
+        context:
+          - check-license
+          - CheckDeadLinks
+          - Build
+          - Spark
+          - Build (8)
+          - Build (11)
```

Comment:
I think we should add the final step to determine all versions build success, it will make our CI rubust because we may add other JDK supported. You could find examples inhttps://github.com/apache/dolphinscheduler/blob/dev/.github/workflows/e2e.yml#L141-L154. Others look good to me

Reply:
Thx, done,PTAL